charts/vela-core/templates/kubevela-controller.yaml

```diff
@@ -121,6 +122,21 @@ spec:
             - containerPort: {{ .Values.webhookService.port }}
               name: webhook-server
               protocol: TCP
+            - containerPort: {{ .Values.healthCheck.port }}
+              name: healthz
+              protocol: TCP
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: healthz
+            initialDelaySeconds: 90
+            periodSeconds: 5
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: healthz
```

Comment:
what does "port: healthz" mean?

Reply:
I think it's brevity for "health check" as [pkg healthz](https://github.com/kubernetes/apiserver/blob/master/pkg/server/healthz/healthz.go) implies.